plasmapy/formulary/collisions.py

```diff
@@ -2214,13 +2259,16 @@ def __init__(
         self.test_particle = test_particle
         self.field_particle = field_particle
         self.v_a = v_a
-        self.T_a = (
+        self._v_a = v_a if v_a is not None else thermal_speed(T_a, test_particle)
+        self.T_a = T_a
```

Comment:
Add a comment about what _v_a and _T_a are. 

It seems T_a and v_a are the input directly as the user gave it, and the private versions are converted using thermal speed? To avoid confusion maybe the private ones should just be variables and re-computed wherever they are needed. 

Actually, maybe the solution is to distinguish between v_a and v_T_a everywhere, as you have done for v_T_b in the mean thermal velocity function! There are some cases where they are the same, but generally the distinction is important.

Reply:
Good call on converting _v_a to v_T_a, that definitely clarifies things. Do you have any suggestions for a new name for _T_a?

Reply:
Where do you use _T_a? I don't see that anymore. 

I like what you have now with _v_a and v_T_a. All I would do is add a switch in the following functions: 

Lorentz_collision_frequency
x

that uses v_a if provided but otherwise uses v_T_a (maybe add a warning that prints out saying you are assuming the relative velocity between the particles is the test particle thermal speed)? 


Not in `mean_thermal_velocity` because there you explicitly want v_T_a even if v_a != 0. So in that function, if T_a is not set then an error should be raised.